src/main/java/filter/expression/QualifierType.java

```diff
@@ -31,6 +32,27 @@
         return Collections.unmodifiableMap(aliases);
     }
 
+    // Ignore Qualifiers that expects a string where any user input would be valid
+    private static Map<QualifierType, String> initialiseValidInputs() {
+        Map<QualifierType, String> validInputs = new HashMap<>();
+        validInputs.put(ID, "a number or a number range");
+        validInputs.put(UPDATED, "a number or  a number range");
+        validInputs.put(STATE, "\"open\" or \"closed\"");
+        validInputs.put(HAS, "\"label\", \"milestone\", or \"assignee\"");
+        validInputs.put(NO, "\"label\", \"milestone\", or \"assignee\"");
+        validInputs.put(IN, "\"title\" or \"body\"");
+        validInputs.put(TYPE, "\"issue\" or \"pr\"");
+        validInputs.put(CREATED, "a date or a date range");
+        validInputs.put(REPO, "repo id");
+        
+        //Show only most common inputs to safe space
```

Comment:
Typo here ("save space"). Also, aren't these all the keywords? Do you mean the frequently-used ones are ordered first?